packages/server/lib/browsers/electron.ts

```diff
@@ -146,9 +146,10 @@ export = {
       onCrashed () {
         const err = errors.get('RENDERER_CRASHED')
 
-        errors.log(err)
```

Comment:
by default 'onError' is defined in open mode, but it was defined as an empty function. this would lead to duplicate logging of errors in run mode. I modified the default onError to log errors and removed this call to remove the duplicate logs.